templates/search.html

```diff
@@ -1,31 +1,27 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group"><!-- wp:search /-->
+<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|70","left":"var:preset|spacing|70"}}}} -->
+<div class="wp-block-columns"><!-- wp:column {"width":"72%","layout":{"type":"constrained"}} -->
+<div class="wp-block-column" style="flex-basis:72%"><!-- wp:heading {"level":1} -->
+<h1>SEARCH</h1>
```

Comment:
We should translate all the text

Reply:
Fixed [be5e087](https://github.com/Automattic/sensei-theme/pull/36/commits/be5e087adc08d44d0936a995c65d453f94896430) and  [432f8b6](https://github.com/Automattic/sensei-theme/pull/36/commits/432f8b6047b1a9ae517938295e59b0ffb2026090)

Reply:
Fixed here   [be5e087](https://github.com/Automattic/sensei-theme/pull/36/commits/be5e087adc08d44d0936a995c65d453f94896430) and [432f8b6](https://github.com/Automattic/sensei-theme/pull/36/commits/432f8b6047b1a9ae517938295e59b0ffb2026090)